extensions/azurePublishNew/src/recoilModel/dispatchers/resourceConfigurationDispatcher.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
```

Comment:
Consider trying to separate the overall resource configuration (aka publish profile config) from the wizard.  The dispatcher could abstract the load and save of the publish profile as a whole, then the individual pieces could be in separate recoil atoms used during the wizard, and at the end you save back to the profile.  Kind of if the profile was a file on disk and the in memory was a bunch of variables.